packages/components/molecules/f-restaurant-card/src/components/subcomponents/IconText.vue

```diff
@@ -98,7 +102,15 @@ export default {
     }
 }
 
-.c-restaurantCard-iconText-icon--green {
-    fill: $color-green;
+.c-restaurantCard-iconText-icon--colorSupportPositive {
+    path {
+        fill: $color-support-positive;
+    }
+}
+
+.c-restaurantCard-iconText-icon--colorSupportInfo {
```

Comment:
As this is duped for both the icon and the icon text I would simplfy the class and use on both. 

eg. `c-restaurantCard-iconText--colorSupportInfo`.

or atleast combine the classnames do not as much dupe? but this might not be needed due to my first comment

Reply:
yeah I like this! just updated